src/acceleration/config/AccelerationConfiguration.cpp

```diff
@@ -359,7 +374,7 @@ void AccelerationConfiguration::addCommonIQNSubtags(xml::XMLTag &tag)
                              "Please note that a QR1 is based on Given's rotations whereas QR2 uses "
                              "modified Gram-Schmidt. This can give different results even when no columns "
                              "are filtered out.");
-  XMLAttribute<double> attrSingularityLimit(ATTR_SINGULARITYLIMIT, 1e-16);
+  XMLAttribute<double> attrSingularityLimit(ATTR_SINGULARITYLIMIT, 1e-2);
```

Comment:
I am not sure if we should change it here as well. 1e-2 is not a good default for QR1. 1e-16 is in a way as it means filter is off.